site/src/pages/CreateTemplatePage/CreateTemplateForm.tsx

```diff
@@ -224,6 +224,18 @@ export const CreateTemplateForm: FC<CreateTemplateFormProps> = ({
   const { t } = useTranslation("createTemplatePage")
   const { t: commonT } = useTranslation("common")
 
+  useEffect(() => {
+    if (error) {
+      window.scrollTo(0, 0)
+    }
+  }, [error])
+
+  useEffect(() => {
+    if (jobError) {
+      window.scrollTo(0, document.body.scrollHeight)
+    }
+  }, [logs])
```

Comment:
I took a look - I think this is pretty good. The only jittering I saw locally was scrolling to the build logs, and this is only because the build logs are expanding as we scroll. I wouldn't worry about this right now.